main.py

```diff
@@ -0,0 +1,103 @@
+""" SHEST FASTAPI SERVER """
+
+from cryptography.fernet import Fernet
+from fastapi import FastAPI, Request
+from os import popen
+from uvicorn import run
+from fastapi.responses import StreamingResponse
+
+
+app = FastAPI()
+
+
+def load_key():
+    """Load the previously generated key"""
```

Comment:
Space must be between the quotes.
```suggestion
    """ Load the previously generated key """
```